docs/configuration/http_filters/router_filter.rst

```diff
@@ -48,10 +48,13 @@ timeout, e.g., early exit. This is set on internal requests and is either taken
 x-envoy-max-retries
 ^^^^^^^^^^^^^^^^^^^
 
-If a retry policy is in place, setting this header controls the number of retries that Envoy will
-perform on the request's behalf. If the header is not specified and there is no :ref:`route
-configuration <config_http_conn_man_route_table_route_retry>` Envoy will perform a single retry. A
-few notes on how Envoy does retries:
+If a :ref:`retry policy <config_http_conn_man_route_table_route_retry>` is in place, Envoy will default to retrying one 
+time unless explicitly specified. You can explicitly set the number of retries either in the 
+:ref:`route retry config <config_http_conn_man_route_table_route_retry>`  or by using this header. If you don't
```

Comment:
nit: please rephrase to avoid "you" in the new text. E.g., "The number of retries can be explicitly set either ..."